SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -47,6 +48,21 @@ bool ComponentFactory::FillSchemaObject(Schema_Object* ComponentObject, UObject*
 			const FRepLayoutCmd& Cmd = Changes.RepLayout.Cmds[HandleIterator.CmdIndex];
 			const FRepParentCmd& Parent = Changes.RepLayout.Parents[Cmd.ParentIndex];
 
+#if TRACE_LIB_ACTIVE
+			if (LatencyTracer)
+			{
+				TraceKey TraceId = LatencyTracer->GetAndUntrackTrace(Object, Cmd.Property);
+				if (TraceId == USpatialLatencyTracer::InvalidTraceKey)
+				{
+					// Possibly sending a nested property? 
```

Comment:
```suggestion
					// Check for sending nested property
```

Reply:
Thoughts on this change?

Reply:
Sorry thought I already committed this. Like it